### PR TITLE
Postgresql branch

### PR DESCRIPTION
**Pull request by sibsmc**
Would like to set up Postgresql database for Ruby Backend. 
In the past we have used a local SQLite database and avoided using Postgresql to make setting up the backend easier. However we would now like to deploy a version of the backend to Heroku so that the frontend can use the API's without needing to install Ruby on their local machine.

### CODE REVIEW

**Comment by sibsmc** [comment]
Please don't merge until build passes

File: server/.#.env [hunk 1]
```diff
@@ -0,0 +1 @@
+siobhanmcloughlin@Siobhans-MBP.h.drongo.info.525
```
Comment (olleolleolle) [inline server/.#.env]: Perhaps this file was not meant to go into the repo?

File: server/config/database.yml [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
-# re-generated from your development database when you run 'rake'.
-# Do not set this db to the same as development or production.
 test:
-  <<: *default
+  #<<: *default
+  adapter: sqlite3
   database: db/test.sqlite3
+  pool: 5
+  timeout: 5000
+  #database: <%= ENV['POSTGRES_TEST_DB'] %>
```
Comment (olleolleolle) [inline server/config/database.yml]: Perhaps we can drop these commented-out lines?

**Comment by sibsmc** [comment]
Thanks for approving!

**Pull request merged**